PluginBuilder/Data/Scripts/14.PluginRatingReviews.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS plugin_reviews (
+  id              BIGSERIAL PRIMARY KEY,
+  plugin_slug     TEXT        NOT NULL,
+  user_id         text        NOT NULL,
+  rating          INT         NOT NULL CHECK (rating BETWEEN 1 AND 5),
+  body            TEXT,
+  plugin_version  INT[],
+  helpful_voters  JSONB       NOT NULL DEFAULT '{}'::jsonb, -- {"<userId>": true|false}
+  created_at      TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  updated_at      TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+
+  UNIQUE (plugin_slug, user_id),
+
+  CONSTRAINT fk_plugin_reviews_plugin
```

Comment:
An unhandled exception occurred while processing the request.
PostgresException: 42703: column "title" of relation "plugin_reviews" does not exist

when tryiong to submit a revieew